site/docs/relations/physical_relations.md

```diff
@@ -206,6 +206,25 @@ The streaming aggregate operation leverages data ordered by the grouping express
 | Measures         | A list of one or more aggregate expressions. Aggregate expressions ordering requirements must be compatible with expected ordering. | Optional, required if no grouping sets. |
 
 
+## Expand Operation
+
+The expand operation creates duplicates of input records based on ExpandFields. For each ExpandField the
+expressions specific to that grouping are evaluated and the result is added to the input record in a manner
+similar to a project operation. 
+
+| Signature            | Value                                                                               |
+| -------------------- | ----------------------------------------------------------------------------------- |
+| Inputs               | 1                                                                                   |
+| Outputs              | 1                                                                                   |
+| Property Maintenance | Distribution and ordering are not maintained.                                       |
```

Comment:
I would expect distribution to be maintained for columns that are duplicated (not switching)